packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -25,18 +75,52 @@ export class FluidAppInsightsLogger implements ITelemetryBaseLogger {
 	 * or else logging will not occur.
 	 */
 	private readonly baseLoggingClient: ApplicationInsights;
+	private readonly config: FluidAppInsightsLoggerConfig;
+	private readonly filters: TelemetryFilter[] = [];
```

Comment:
Nit: since we're storing the full config anyways, this could just be a getter property that returns `this.config.filters`

Reply:
Or just use `this.config.filters` directly where we need them, it's only 2 places.

Reply:
There are parts of this design that make more sense once namespaces are introduced... I will update it to drop the "filters" member variable for now.